engines/wintermute/base/font/base_font_bitmap.cpp

```diff
@@ -509,6 +518,11 @@ bool BaseFontBitmap::persist(BasePersistenceManager *persistMgr) {
 		persistMgr->getBytes(_widths, sizeof(_widths));
 	}
 
+#ifdef ENABLE_FOXTAIL
+	if (!persistMgr->getIsSaving()) {
+		_minimizeSpacing = BaseEngine::instance().getGameId() == "foxtail";
```

Comment:
Am I understanding this right? minimizeSpacing is always true in foxtail?

Reply:
Yes. FoxTail has a globally enabled hack for making spaces between character and spaces between lines smaller than it is according to FONT settings. I added _minimizeSpacing to avoid checking if game is FoxTail on each BaseFontBitmap::getCharWidth(). 

Now when I think about it, maybe we should just make some globally avaliable bool at BaseEngine::instance() and remove _minimizeSpacing from BaseFontBitmap completely?

Reply:
> maybe we should just make some globally avaliable bool at BaseEngine::instance() and remove _minimizeSpacing from BaseFontBitmap completely?

I'd say it's a good idea.

Could this be the time to introduce some general way of marking _features_ (including "bugfeatures") required/expected by a WME game?